### PR TITLE
ci: release & upload UIs to artifactory without maven

### PR DESCRIPTION
**Pull request by ThibaudAV**
**Issue**

https://github.com/gravitee-io/issues/issues/5462

**Description**

Improve release for Console, Portal & Backend
Remove use of gravitee-io orb

**Additional context**
image

https://app.circleci.com/pipelines/github/gravitee-io/gravitee-api-management/5672/workflows/0df361f2-d813-4ddc-9b28-aacc3e1d54c8

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-dzxxwgzemf.chromatic.com)

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-ecrgzwdpjg.chromatic.com)

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/5462-remove-maven-ui-projects/index.html)
_Notes_: The deployed app is linked to the management API of APIM master. (Same login and password as APIM master)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -484,6 +485,42 @@ jobs:
...
                   path: << parameters.apim-ui-project >>/coverage
 
+    webui-publish-artifactory:
+        parameters:
+            apim-ui-project:
+                type: string
+                default: ""
+                description: the name of the UI project to build
+        environment:
+            DRY_RUN: << pipeline.parameters.dry_run >>
```
Comment (gaetanmaisse) [inline .circleci/config.yml]: Needed?

File: .circleci/config.yml [hunk 2]
```diff
@@ -1234,17 +1340,18 @@ workflows:
...
+            #           - Commit and prepare next version
+            #       context: cicd-orchestrator
+            #       pre-steps:
+            #           - secrethub/env-export:
+            #                 secret-path: graviteeio/cicd/graviteebot/infra/zip-bundle-server/clever-cloud-s3/s3cmd/aws_access_key
+            #                 var-name: AWS_ACCESS_KEY_ID
+            #           - secrethub/env-export:
+            #                 secret-path: graviteeio/cicd/graviteebot/infra/zip-bundle-server/clever-cloud-s3/s3cmd/aws_secret_key
+            #                 var-name: AWS_SECRET_ACCESS_KEY
+
```
Comment (gaetanmaisse) [inline .circleci/config.yml]: To remove?

File: .circleci/config.yml [hunk 3]
```diff
@@ -2,7 +2,6 @@ version: 2.1
 
 orbs:
     slack: circleci/slack@4.4.4
-    gravitee: gravitee-io/gravitee@1.0.44
```
Comment (gaetanmaisse) [inline .circleci/config.yml:5]: :youhou:

File: .circleci/config.yml [hunk 4]
```diff
@@ -855,20 +780,15 @@ jobs:
             - run:
                   name: Update maven dependencies versions from properties
                   command: |
-                      mvn -Duser.home=${HOME} -s ./release.settings.xml -B -U versions:update-properties -Dmaven.version.rules.serverId=artifactory-gravitee-releases -Dincludes="io.gravitee.*:*" -Dexcludes="io.gravitee.policy:*,io.gravitee.connector:*,io.gravitee.resource:*,io.gravitee.service:*,io.gravitee.fetcher:*,io.gravitee.tracer:*,io.gravitee.repository:*,io.gravitee.reporter:*,io.gravitee.cockpit:*,io.gravitee.discovery:*,io.gravitee.notifier:*,com.graviteesource.notifier:*,com.graviteesource.policy:*" -DallowMajorUpdates=false -DallowMinorUpdates=false -DallowIncrementalUpdates=true -DgenerateBackupPoms=false -T 2C
+                      mvn versions:update-properties --batch-mode -Dmaven.version.rules.serverId=artifactory-gravitee -Dincludes=io.gravitee.*:* -DallowMajorUpdates=false -DallowMinorUpdates=false -DallowIncrementalUpdates=true -DgenerateBackupPoms=false --settings .gravitee.settings.xml --update-snapshots
```
Comment (gaetanmaisse) [inline .circleci/config.yml]: exclude are not needed anymore? (event just in case)

**Review by gaetanmaisse — APPROVED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (ThibaudAV) [reply .circleci/config.yml]: done in next commit. Disabled here for test

File: .circleci/config.yml (hunk 4 above)
Comment (ThibaudAV) [reply .circleci/config.yml]: i think yes. unwanted change. 
i've add a fixup

**Review by gcusnieux — APPROVED** [review]

**Review by gaetanmaisse — APPROVED** [review]

**Pull request merged**